libs/UIHelpers/transfer_function_widget.cpp

```diff
@@ -439,52 +478,49 @@ void TransferFunctionWidget::draw_histogram()
 
   ImGui::InvisibleButton("hstg_canvas", canvas_size);
 
-
   // Draw the alpha control points, and build the points for the polyline
   // which connects them
 
-
-  //Code to Draw histogram in the UI
-  for (int i = 0; i < current_histogram.size(); i++) {
+  // Code to Draw histogram in the UI
+  for (int i = 0; i < current_histogram.size(); i++)
+  {
     vec2f lp = vec2f(((float)i) / current_histogram.size(), 0.0f);
     vec2f hp = vec2f(((float)i + 1.0f) / current_histogram.size(), current_histogram[i]);
     draw_list->AddRectFilled(lp * view_scale + view_offset, hp * view_scale + view_offset, 0x77777777);
   }
 
-
   draw_list->PopClipRect();
-
-
-
-
 }
 
 void TransferFunctionWidget::update_gpu_image()
 {
   GLint prev_tex_2d = 0;
   glGetIntegerv(GL_TEXTURE_BINDING_2D, &prev_tex_2d);
 
-  if (colormap_img == (GLuint)-1) {
+  if (colormap_img == (GLuint)-1)
+  {
     glGenTextures(1, &colormap_img);
     glBindTexture(GL_TEXTURE_2D, colormap_img);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_CLAMP_TO_EDGE);
     glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_CLAMP_TO_EDGE);
   }
-  if (colormap_changed) {
+  if (colormap_changed)
+  {
     glBindTexture(GL_TEXTURE_2D, colormap_img);
     glTexImage2D(GL_TEXTURE_2D,
-      0,
-      GL_RGBA,
-      current_colormap.size() / 4,
-      1,
-      0,
-      GL_RGBA,
-      GL_UNSIGNED_BYTE,
-      current_colormap.data());
+                 0,
+                 GL_RGBA,
+                 current_colormap.size() / 4,
+                 1,
+                 0,
+                 GL_RGBA,
+                 GL_UNSIGNED_BYTE,
+                 current_colormap.data());
```

Comment:
Since the function call is spanning multiple lines I would move the closing parenthesis to a new line that's aligned with the `glTexImag2D` call itself. I may not having lined things up correctly in the suggestion so I wouldn't commit it directly.
```suggestion
                 current_colormap.data()
   );
```

Reply:
https://github.com/brown-ccv/VR-Volumeviewer/issues/49